db/migrate/20240314171529_create_adopter_foster_accounts.rb

```diff
@@ -0,0 +1,10 @@
+class CreateAdopterFosterAccounts < ActiveRecord::Migration[7.0]
```

Comment:
Instead of creating a new table, we could just rename the `adopter_accounts` table to `adopter_foster_accounts`